src/static/webcomponents/topdowncompass/topdowncompass.html

```diff
@@ -0,0 +1,52 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<polymer-element name="rov-topdowncompass">
+  <template>
+    <link rel="stylesheet" href="/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/topdowncompass.css">
+
+    <style type="text/css">
+    </style>
+
+    <div id="compass" class="topdowncompass pull-left">
+      <div id="needle" class="compassNeedle"></div>
+    </div>
+
+  </template>
+  <script>
+    (function() {
+
+      Polymer({
+        ready: function () {
+          var compass = this;
+          window.cockpit.rov.on('plugin.navigationData.data', function(data) {
```

Comment:
Note to have a discussion to sync up on the end state for the webcomponents. Having the child components have explicit references to the parents is generally a bad practice for components but is clearly a result of bundling the prior pattern up.  Lets be in sync on if this is a stop gap or end state that we are at.